index.bs

```diff
@@ -12014,19 +12014,26 @@ However, for [=legacy platform objects=],
 properties on the object must be
```

Comment:
Would be great to add a small blurb here that this is done by the [[OwnPropertyKeys]] method, e.g: 

```
This document does not define a complete property enumeration order
for [=platform objects=] implementing [=interfaces=]
(or for <a href="#es-exception-objects">platform objects representing exceptions</a>).
However, it does for [=legacy platform objects=] by defining the \[[OwnPropertyKeys]] method as follows:
```